segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/filesystem/FileSystemStorageConfig.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.storage.impl.filesystem;
+
+import io.pravega.common.util.ConfigBuilder;
+import io.pravega.common.util.ConfigurationException;
+import io.pravega.common.util.Property;
+import io.pravega.common.util.TypedProperties;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Configuration for the NFS Storage component.
+ */
+@Slf4j
+public class FileSystemStorageConfig {
+    //region Config Names
+
+    public static final Property<String> ROOT = Property.named("filesystemRoot", "/fs/");
```

Comment:
this will look like `filesystem.filesystemRoot`. I would drop the `filesystem` from the property name. The prefix indicates it's part of the `filesystem` namespace.